scripts/install_obsolete_jsoncpp_1_7_4.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env sh
+set -e
+
+TEMPDIR=$(mktemp -d)
+JSONCPP="jsoncpp-1.7.4"
+
+cd $TEMPDIR
```

Comment:
Please create a subshell here, i.e. use `(` - we should not clutter the cwd of the shell running the script.